### PR TITLE
Add liquidity mining deployment task

### PR DESCRIPTION
**Pull request by manhlx3006**

### CODE REVIEW

File: deployment/liquidityMining/deployLiquidityMining.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
+require('@nomiclabs/hardhat-ethers');
+const inquirer = require('inquirer');
```
Comment (Anyhowclick) [inline deployment/liquidityMining/deployLiquidityMining.js:2]: can remove

File: deployment/liquidityMining/deployLiquidityMining.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  async (taskArgs, hre) => {
+    parseInput(configParams);
+
+    const BN = ethers.BigNumber;
+    const [deployer] = await ethers.getSigners();
+    deployerAddress = await deployer.getAddress();
+    console.log(`Deployer address: ${deployerAddress}`)
+
+    let outputData = {};
+    gasPrice = new BN.from(10).mul(new BN.from(10).pow(new BN.from(9)));
```
Comment (Anyhowclick) [inline deployment/liquidityMining/deployLiquidityMining.js:41]: maybe take in as input argument

**Review by ducquangkstn — APPROVED** [review]

File: deployment/liquidityMining/deployLiquidityMining.js (hunk 1 above)
Comment (manhlx3006) [reply deployment/liquidityMining/deployLiquidityMining.js]: removed

File: deployment/liquidityMining/deployLiquidityMining.js (hunk 2 above)
Comment (manhlx3006) [reply deployment/liquidityMining/deployLiquidityMining.js]: Added as input param, in gweis

**Pull request merged**